tests/importer/test_importer.py

```diff
@@ -296,25 +307,14 @@ def setUp(self):
         # Setup mocked APIs
         self.mock_ingest_api = MagicMock(spec=IngestApi)
 
-    def test_import_file_throwing_submission_error(self):
-        # given:
-        exception = ingest.importer.submission.Error(
-            'http://unittest.importer.ingest.data.humancellatlas.org/Error',
-            'Error thrown for Unit Test')
-        exception_json = SubmissionError(str(exception)).getJSON()
-        self.import_file_with_exception(exception, exception_json)
-
     def test_import_file_throwing_exception(self):
         # given:
         exception = Exception('Error thrown for Unit Test')
         exception_json = ImporterError(str(exception)).getJSON()
-        self.import_file_with_exception(exception, exception_json)
-
-    def import_file_with_exception(self, exception, exception_json):
-        # given:
         importer = XlsImporter(ingest_api=self.mock_ingest_api)
         importer._generate_spreadsheet_json = MagicMock(side_effect=exception)
         importer.logger.error = MagicMock()
+
```

Comment:
could add a test case for importing with errors

Reply:
Kind of covered by `test_do_import_multiple_projects` & `test_do_import_no_projects` but I'll take a look.

Reply:
cool, just ignore this then. :)